### PR TITLE
Add OWNERS files generated from commit log.

### PR DESCRIPTION
**Pull request by tobie**
As [announced in public-test-infra@](https://lists.w3.org/Archives/Public/public-test-infra/2016JanMar/0001.html), we've added support for OWNERS file to the wpt repo.

This allows you to turn off notifications for the wpt repo ("NOT WATCHING" status) and be automatically tagged in PR which touch code you own.

In order to accelerate the setup, I've generated OWNERS files for most repos based on commit history. _(Note OWNERS file can be added to nested directories, but I voluntarily kept it to top-level dirs this time.)_

I've set this PR as a branch on the main repo, so you should be able to push changes to it directly (updating this PR in the process), if you want to remove or add your name somewhere.

You're in a least one of the OWNERS file if your name is below:

@acolwell, @AFBarstow, @alsemenov, @alvestrand, @andreastt, @annevk, @anssiko, @aogilvie, @AutomatedTester, @ayg, @bethge, @birtles, @bit, @bzbarsky, @caitp, @chaals, @chrislo, @ChrisParis, @chunywang, @cptvitamin, @cvrebert, @cynthia, @deepak-sa, @deniak, @dontcallmedom, @dumbmatter, @ecoal95, @ehsan, @emalasky, @EvgenyAgafonchikov, @fmarier, @foolip, @frewsxcv, @gsnedders, @hallvors, @haoxli, @happyoungj, @heycam, @hillbrad, @ibelem, @inexorabletash, @jacobrossi, @jdm, @jonathanKingston, @jtangelder, @jungkees, @Jxck, @kangxu, @kburnik, @kojiishi, [[...]](#issuecomment-176988364)

Note: I removed @jgraham and @Ms2ger as they're pretty much all over the place.

### CODE REVIEW

**Comment by tobie** [comment]
/cc:  @koustuvsinha, @kristijanburnik, @lukeis, @luser, @Manishearth, @mastahyeti, @mathiasbynens, @metromoxie, @mikewest, @omo, @phoglund, @plehegar, @rniwa, @ronkorving, @rubys, @scottgonzalez, @sgrekhov, @shishimaru, @shs96c, @sideshowbarker, @siusin, @sizuhiko, @smola, @staktrace, @Steditor, @tagawa, @tidoust, @tomalec, @valenting, @Velmont, @Volker-E, @vyv03354, @Wafflespeanut, @xiaojunwu, @xinliux, @Yaffle, @YuichiNukiyama, @yunxiaoxie, @yutak, @zcorpan, @zhaozihao, @zqzhang, @jgraham and @Ms2ger.

**Comment by bzbarsky** [comment]
If I don't want to be tagged for PRs, can I just ask to be removed from the OWNERS files?

**Comment by tobie** [comment]
@bzbarsky absolutely. Done.

File: webdriver/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@andreastt
+@lukeis
+@AutomatedTester
+@shs96c
+@emalasky
```
Comment (AutomatedTester) [inline webdriver/OWNERS]: We havent had emalasky in WG meetings so not sure why he is here.

**Comment by bzbarsky** [comment]
@tobie Thanks!

File: webdriver/OWNERS (hunk 1 above)
Comment (tobie) [inline webdriver/OWNERS]: He probably merged a commit that touched the webdriver test suite at some point.

**Comment by annevk** [comment]
@tobie can you remove me too? Happy to get a ping now and then, but not going to sign up as owner.

**Comment by tobie** [comment]
@annevk done.

File: mediacapture-streams/OWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+@dontcallmedom
```
Comment (alvestrand) [inline mediacapture-streams/OWNERS]: can you add me here too? I'm happy to help watch the repo.

**Comment by kburnik** [comment]
Hi,

I had a commit or two with my personal account by accident. Could you
please remove @kburnik ? The other one: @kristijanburnik can stay. :-)

Thanks.

**Comment by tobie** [comment]
@kburnik done.

File: mediacapture-streams/OWNERS (hunk 2 above)
Comment (tobie) [inline mediacapture-streams/OWNERS]: done

**Comment by cvrebert** [comment]
@tobie Remove me entirely please.

**Comment by valenting** [comment]
Please remove me too. Thanks!

**Comment by vyv03354** [comment]
Please remove me too.

**Comment by tobie** [comment]
@vyv03354 I don't see you in there. Not sure why I pinged you in the first place. Apologies.

**Comment by Ms2ger** [comment]
Please add me to
- DOMEvents
- FileAPI
- WebCryptoAPI
- XMLHttpRequest
- docs
- dom
- domparsing
- svg
- typedarrays
- webmessaging
- websockets
- webstorage
- workers

**Comment by jdm** [comment]
Please add me to FileAPI, XMLHttpRequest, websockets, webstorage, workers, html, dom, cors, domparsing, fetch, geolocation-API, and http.

**Comment by tobie** [comment]
@jdm, @Ms2ger: As you're both collaborators, you can just push to the [add-owners](https://github.com/w3c/web-platform-tests/tree/add-owners) branch on [w3c/web-platform-tests](https://github.com/w3c/web-platform-tests/tree/add-owners).

**Comment by inexorabletash** [comment]
`service-workers/cache-storage` probably merits its own OWNERs. me, @wanderview and ... ?

**Comment by tobie** [comment]
@inexorabletash added an owners file for cache-storage. Feel free to update it.

**Comment by tobie** [comment]
The beauty of this system is that it's easily modifiable so I don't think we should get too hung-up on getting this PR 100% perfect before shipping it. I think what's more important is that people see the overall value this solution can provide and agree to use it (or at least to give it a try).

**Comment by foolip** [comment]
I see no `html/OWNERS`, is that an oversight? `html/semantics/embedded-content/media-elements/` is what I'm most familiar with, of these tests.

**Comment by jdm** [comment]
It was added in a previous PR.

**Comment by jgraham** [comment]
I think we should just land this and sort out changes later.

**Pull request merged**